### PR TITLE
Adds OSGi related utility classes.

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: src/main/java/org/jitsi/osgi/EventHandlerActivator.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/*
+ * Jicofo, the Jitsi Conference Focus.
```
Comment (gpolitis) [inline src/main/java/org/jitsi/osgi/EventHandlerActivator.java]: Not sure what we're supposed to use for Jicoco..

Comment (paweldomas) [inline src/main/java/org/jitsi/osgi/EventHandlerActivator.java]: Oh yes will fix that :) Thanks for noticing !

Comment (gpolitis) [inline src/main/java/org/jitsi/osgi/EventHandlerActivator.java]: Would you like me to wait for a fixed header to merge this?

Comment (paweldomas) [inline src/main/java/org/jitsi/osgi/EventHandlerActivator.java]: Yes, it should be there now !

**Pull request merged**